ftl/core/browsing.ftl

```diff
@@ -8,6 +8,7 @@ browsing-answer = Answer
 browsing-any-cards-mapped-to-nothing-will = Any cards mapped to nothing will be deleted. If a note has no remaining cards, it will be lost. Are you sure you want to continue?
 browsing-any-flag = Any Flag
 browsing-average-ease = Average Ease
+browsing-average-interval = Average Interval
```

Comment:
Those are a bit long compared to the strings in their columns. Should we use "Avg." or "⌀"?